simulator/src/replay/cmd_run.rs

```diff
@@ -150,3 +90,90 @@ impl Run {
         Ok(())
     }
 }
+
+pub fn print_progress(duration: Duration, new_version: u64, new_root: Hash) {
```

Comment:
Could probably keep this function and the other functions private.

Reply:
Will make it mod private.